AsyncDisplayKit/ASTextNode.mm

```diff
@@ -749,6 +749,33 @@ - (UIImage *)placeholderImage
 
 #pragma mark - Touch Handling
 
+-(BOOL)pointInside:(CGPoint)point withEvent:(UIEvent *)event
+{
+  if (!_passthroughNonlinkTouches) {
+    return [super pointInside:point withEvent:event];
+  }
+
+  NSRange range = NSMakeRange(0, 0);
+  NSString *linkAttributeName = nil;
+  BOOL inAdditionalTruncationMessage = NO;
+
+  id linkAttributeValue = [self _linkAttributeValueAtPoint:point
+                                             attributeName:&linkAttributeName
+                                                     range:&range
+                             inAdditionalTruncationMessage:&inAdditionalTruncationMessage];
+
+  NSUInteger lastCharIndex = NSIntegerMax;
```

Comment:
Yeah that does look a little suspicious. This is a straight copy and paste from `touchesBegan:withEvent:`, so I didn't really look into how it worked too much. Suggest we merge this PR then fix in both places as a separate PR? (Goal would be to clean-up and merge the logic into one place.)